src/main/java/org/rust/lang/doc/psi/RustDocElementTypes.java

```diff
@@ -0,0 +1,13 @@
+package org.rust.lang.doc.psi;
+
+public interface RustDocElementTypes {
+    RustDocTokenType DOC_TEXT = new RustDocTokenType("<DOC_TEXT>");
+    RustDocTokenType DOC_DECO = new RustDocTokenType("<DOC_DECO>");
```

Comment:
Have in mind, that these guys are not present in the actual PSI, they are only visible to the highlighter (currently)